dashboard/src/errors/QueryAbortedError.js

```diff
@@ -0,0 +1,10 @@
+import ExtendableError from "es6-error";
+
+class QueryAbortedError extends ExtendableError {
```

Comment:
Without a concrete use case at the moment it's difficult for me to envision when we might throw this. May be worth documenting either here or in the `Query` component.

Reply:
Concrete case coming Soon™...